modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ListEntries.es.js

```diff
@@ -21,11 +22,12 @@ import {Loading} from '../../components/loading/Loading.es';
 import useDataListView from '../../hooks/useDataListView.es';
 import useEntriesActions from '../../hooks/useEntriesActions.es';
 import usePermissions from '../../hooks/usePermissions.es';
+import useQuery from '../../hooks/useQuery.es';
 import {getLocalizedUserPreferenceValue} from '../../utils/lang.es';
 import NoPermissionEntry from './NoPermissionEntry.es';
 import {buildEntries, navigateToEditPage} from './utils.es';
 
-export default function ListEntries() {
+export default withRouter(({history}) => {
```

Comment:
Hey guys @interaminense @kevenleone , is this necessary?

I think the history property is already injected into the component by the route. You can see how is being retrieved on this file from the workflow side.

Reply:
@albinojunior, Do you mean I can get the `history` from props?

Reply:
Yes. @interaminense .

Reply:
I'll take a look at this